src/tests/models.py

```diff
@@ -11,10 +10,11 @@
 logger = logging.getLogger(__name__)
 
 
-@register_oapif_viewset(crs=2056)
-class Point_2056_10fields(ComputedFieldsModel):
+class BaseModel10Fields(models.Model):
```

Comment:
```suggestion
class BaseModelWithTenFields(models.Model):
```

Slighthy more verbose, but more explicit. I initialy thought it's something like `i18n` or `l10n`